What's new in v4 (#671)
akeneo/pim-helpcenter: content/updates/data-quality-insights/1-data-quality-dashboard.md

> # A data quality dashboard 

Reviewer: ```suggestion
# Monitor your data quality in a dashboard 
```

> With the Data Quality Insights feature, you can easily measure the quality of your product data and see at a glance how it measures up in a dedicated dashboard.

Reviewer: ```suggestion
With the Data Quality Insights feature, you can easily **measure the quality of your product data and monitor it at a glance** in a dedicated dashboard.
```

> This dashboard uses a grading system based on enrichment and consistency. The enrichment axis checks required and optional attributes, while the consistency axis monitors spelling and case formatting.

Reviewer: ```suggestion
This dashboard uses a **grading system based on enrichment and consistency**. The enrichment axis checks required and optional attributes, while the consistency axis monitors spelling and case formatting.
```

> In addition to a global product catalog view, you can focus on the quality of a specific category or family to better prioritize your tasks.

Reviewer: ```suggestion
In addition to a global product catalog view, you can **focus on the quality of a specific category or family to better prioritize your tasks**.
```
